Buildings/Applications/DHC/CentralPlants/Building/package.order

```diff
@@ -0,0 +1 @@
+Building
```

Comment:
The package `Buildings/Applications/DHC/CentralPlants/Building/` should be deleted.

Reply:
I have deleted unnecessary files and folders that were commited before.